fix(docs): update log injection support (#14031)
DataDog/dd-trace-py: docs/configuration.rst

.. ddtrace-configuration-options::

   DD_LOGS_INJECTION:
     type: Boolean
     default: true
     description: Enables :ref:`Logs Injection`. Supported values are ``false``, and ``true``.

>     version_added:

Author: ```suggestion
     version_added:
```

       v0.51.0: |
         Added support for correlating traces to log using the builtin logger. This featrue was disabled by default.
       v3.10.0: |

>          The default value was changed to ``true``. This means that the tracer will inject trace context into logs by default.

Author: ```suggestion
         The default value was changed to ``true``. This means that the tracer will inject trace context into logs when ``ddtrace-runt`` or ``import ddtrace.auto`` is used.
```

>            The default value was changed to ``true``. This means that the tracer will inject trace context into logs by default.

Author: ```suggestion
         The default value was changed to ``true``. This means that the tracer will inject trace context into logs by default.
```

>          The default value was changed to ``true``. This means that the tracer will inject trace context into logs when ``ddtrace-runt`` or ``import ddtrace.auto`` is used.

Reviewer: ```suggestion
         The default value was changed to ``true``. This means that the tracer will inject trace context into logs when ``ddtrace-run`` or ``import ddtrace.auto`` is used.
```
